inc/blacklist.class.php

```diff
@@ -132,7 +132,7 @@ function prepareInputForAdd($input) {
    }
 
 
-   function displaySpecificTypeField($ID, $field = []) {
+   function displaySpecificTypeField($ID, $field = [], $options = []) {
```

Comment:
Sounds useless

Reply:
adding the option array to ```displaySpecificTypeField``` (see ```CommonDropdown.class.php```

allows me to create a ```Socket``` from an asset (ex: ```Computer```)

This behaviors already exist from GLPI  (we can create  ```Change``` from ```Computer``` )

but not for classes which extends ```CommonDropdown````

From this PR
see socket.class.php:607
see socket.form.php:36